Sources/SwiftParser/TokenSpecSet.swift

```diff
@@ -482,6 +482,7 @@ enum AwaitTryMove: TokenSpecSet {
   case _borrowKeyword
   case tryKeyword
   case consumeKeyword
+  case copyKeyword
```

Comment:
This `TokenSpecSet` has long since outgrown its name of `AwaitTryMove`. What do you think about renaming it to `ExpressionPrefix`? I’m open to better naming suggestions.